Frameworks/ImageIO/CGImageSource.mm

```diff
@@ -1718,7 +1718,7 @@ CGImageRef CGImageSourceCreateImageAtIndex(CGImageSourceRef isrc, size_t index,
                                         32,
                                         frameWidth * 4,
                                         colorspaceRgb,
-                                        kCGImageAlphaFirst,
+                                        (kCGImageAlphaLast | kCGBitmapByteOrder32Little),
```

Comment:
That is correct. I initially put it here to be more explicit but for consistency and thoroughness,, I reverted use of kCGBitmapByteOrder32Little here and elsewhere where it should be assumed anyway and retested the code once more.